experiment/simulation/css/main.css

```diff
@@ -1 +1,363 @@
+  background-color: rgb(4, 5, 5);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+.components-list {
+  padding: 2% 5%;
+  text-align: center;
+  height: 100%;
+}
+
+#circuit-board {
+  position: relative;
+  width: 100%;
+  height: 100%;
+  border: 0.2rem solid black;
+}
+
+#working-area {
+  position: absolute;
+  width: 95%;
+  height: 95%;
+  margin: 2%;
+}
+
+.connector-point {
+  position: absolute;
+  z-index: 1;
+  display: block;
+  height: 15px;
+  width: 1px;
+  border-radius: 50%;
+  background-color: grey;
+}
+
+svg.wire path {
+  stroke: #456;
+  stroke-width: 3;
+}
+
+.menu {
+  width: 120px;
+  z-index: 1;
+  box-shadow: 0 4px 5px 3px rgba(0, 0, 0, 0.2);
+  background-color: #fff;
+  position: fixed;
+  display: none;
+  transition: 0.2s display ease-in;
+}
+
+.menu-options {
+  list-style: none;
+  padding: 10px 0;
+  z-index: 1;
+}
+
+.menu-option {
+  font-weight: 500;
+  z-index: 1;
+  font-size: 14px;
+  padding: 10px 40px 10px 20px;
+}
+
+.menu-option:hover {
+  background: rgba(0, 0, 0, 0.2);
+  cursor: pointer;
+}
+
+.component-button{
+  width: 80px;
+  height: 80px;
+  border-radius: 0.75em;
+  border: 0.2rem solid rgb(98, 178, 235);
+  margin: 1rem 0;
+  font-weight: bold;
+  font-size: 0.8rem;
+  text-align: center;
+  color: black;
+  background-color: #29e;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+@media screen and (max-width: 600px) {
+  .component-button {
+    width: 100px;
+    height: 100px;
+    background-size: 100%;
+  }
+}
+
+.component-button::after {
+  display: block;
+  padding-bottom: 100%;
+}
+
+.component-button:hover {
+  border: 0.2rem solid rgb(157, 210, 248);
+  cursor: pointer;
+}
+
+
+
+.success-message {
+  font-size: 1.2rem;
+  color: rgb(31, 170, 31);
+  font-weight: bold;
+}
+
+.failure-message {
+  font-size: 1.2rem;
+  color: rgb(228, 23, 23);
+  font-weight: bold;
+}
+
+
+
+/* for animation */
+#zero-one {
+  position: absolute;
+  top: 0px;
+  left: 320px;
+}
+#one-zero {
+  position: absolute;
+  top: 320px;
+  left: 320px;
+}
+#one-one {
+  position: absolute;
+  top: 320px;
+  left: 0px;
+}
+#path1 {
+  position: absolute;
+  top: 50px;
+  left: 20px;
+}
+
+#path2 {
+  position: absolute;
+  top: 50px;
+  left: 20px;
+}
+
+#path3 {
+  position: absolute;
+  top: 50px;
+  left: 20px;
+}
+
+#path4 {
+  position: absolute;
+  top: 50px;
+  left: 20px;
+}
+
+.square {
+pointer-events: none;
+background-color: black;
+margin: 4px;
+display: inline-block;
+opacity: 0.6;
+border-radius: 5px;
+width: 20px;
+height: 20px;
+}
+.square.blue {
+  left: -10px;
+  top: -10px;
+  background: blue;
+  position: absolute;
+}
+.square.black.xor {
+  pointer-events: none;
+  background-color: black;
+  margin: 4px;
+  display: inline-block;
+  opacity: 1;
+  border-radius: 5px;
+  width: 30px;
+  height: 30px;
+  left: 128px;
+  top: 85px;
+  font-size: 10px;
+  color: white;
+  background: black;
+  position: absolute;
+}.square.black.and {

```

Comment:
Please add a new line for the next selector

Reply:
Modified css file, removed unused classes.